File: web/src/components/editors/prose-editor/nodes/math.ts
import StencilaMath from '../../../nodes/math'
import { executableAttrs, StencilaExecutableView } from './executable'

export const mathAttrs = {
  ...executableAttrs,
  mathLanguage: { default: 'tex' },
  code: { default: '' },
  mathml: { default: '' },
}

export class StencilaMathView<
  Type extends StencilaMath
> extends StencilaExecutableView<Type> {}

File: rust/node-patch/src/math.rs
//! Patching for math related nodes

use stencila_schema::*;

use super::prelude::*;

patchable_struct!(
    MathBlock,
    id,
    math_language,
    code,
    compile_digest,
    errors,
    mathml
);

patchable_struct!(
    MathFragment,
    id,
    math_language,
    code,
    compile_digest,
    errors,
    mathml
);
